### PR TITLE
[Keyboard] Enable RGB Matrix for Corne

### PR DESCRIPTION
**Pull request by drashna**
## Description

Enable RGB Matrix by default, add a default logo for the OLED screens, and a bit of cleanup.

Depends on #13060 or a tweak to define. 

## Types of Changes

- [x] Keyboard (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/rules.mk [hunk 1]
```diff
@@ -17,24 +10,27 @@ BOOTLOADER = caterina
...
 MOUSEKEY_ENABLE = yes       # Mouse keys
-EXTRAKEY_ENABLE = no       # Audio control and System control
+EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no        # Commands for debug and configuration
+COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 MIDI_ENABLE = no            # MIDI controls
```
Comment (fauxpark) [inline keyboards/crkbd/rules.mk:18]:
```suggestion
```

File: keyboards/crkbd/rules.mk [hunk 2]
```diff
@@ -17,24 +10,27 @@ BOOTLOADER = caterina
...
-AUDIO_ENABLE = no           # Audio output on port C6
+AUDIO_ENABLE = no           # Audio output
 UNICODE_ENABLE = no         # Unicode
```
Comment (fauxpark) [inline keyboards/crkbd/rules.mk:20]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/rules.mk [hunk 3]
```diff
@@ -17,24 +10,25 @@ BOOTLOADER = caterina
...
-MIDI_ENABLE = no            # MIDI controls
-AUDIO_ENABLE = no           # Audio output on port C6
-UNICODE_ENABLE = no         # Unicode
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
-RGB_MATRIX_ENABLE = no
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/crkbd/rules.mk:20]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/crkbd/rules.mk [hunk 4]
```diff
@@ -17,24 +10,25 @@ BOOTLOADER = caterina
 #
 BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
 MOUSEKEY_ENABLE = yes       # Mouse keys
-EXTRAKEY_ENABLE = no       # Audio control and System control
+EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no        # Commands for debug and configuration
+COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (fauxpark) [inline keyboards/crkbd/rules.mk:16]:
```suggestion
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Honestly, I am thinking that it may be better to close this and not add rgb matrix to the default. 

Or like the ergodox, add a rgb matrix and rgb light revisions.  but that feels ... dirty.

**Pull request closed without merging**